libraries/botbuilder-dialogs-adaptive/src/input/choiceInput.ts

```diff
@@ -136,4 +132,16 @@ export class ChoiceInput extends InputDialog {
         return `ChoiceInput[${ this.prompt && this.prompt.toString() }]`;
     }
 
+    private determineCulture(dc: DialogContext, opt: FindChoicesOptions = null): string {
```

Comment:
Shouldn't this mirror the C# implementation of [GetCulture()](https://github.com/microsoft/botbuilder-dotnet/blob/46aa02a7875583fed000046f013eeac9f70d41b0/libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/Input/ChoiceInput.cs#L194-L206)

Reply:
I discuss in the description that I've made changes that, if accepted, may require parity pull requests in other repos. In this case, the C# implementation is flawed because it does not include the `FindChoicesOptions` as a potential fallback. Would you like me to remove everything that doesn't match C# and save those changes for later, or is it okay to make this fixes now?